Makefile

```diff
@@ -32,7 +32,7 @@ site-docs: site
 	for f in site/manual/gh*.md; do sed -i.bak -e '/^### SEE ALSO/,$$d' "$$f"; done
 	rm -f site/manual/*.bak
 	git -C site add 'manual/gh*.md'
-	git -C site commit -m 'update help docs' || true
+	git -C site commit --author "cli automation <noreply@github.com>" -m 'update help docs' || true
```

Comment:
These tasks might be run locally by an actual human, so in that case it might be useful to preserve correct attribution there.

Since we need to fix CI only, how about we set these for the step in the release workflow?
```yaml
env:
  GIT_COMMITTER_NAME: cli automation
  GIT_AUTHOR_NAME: cli automation
  GIT_COMMITTER_EMAIL: noreply@github.com
  GIT_AUTHOR_EMAIL: noreply@github.com
```

Reply:
sure. that makes more sense, thanks